cypress/e2e/RoundTrips/TextEditor.feature

```diff
@@ -603,3 +604,30 @@ Feature: Test roundtrip of the application: text editor
     Then I expect '[data-cy="file-tabs-container"] [data-cy="active-tab"]' is 'new_file.tf'
     And  I expect '[data-cy="file_infra/new_file.tf"]' exists
     And  I expect active file content to contain 'provider.*"aws".*{}'
+
+    ## Rename a folder
```

Comment:
Currently, if we rename the active file (with its tab open), the active tab with the old name is closed, but I expected the active tab to still be open with the new file name, what do you think ?

Reply:
Won't do. Seen with @Zorin95670